src/clr-angular/timeline/_timeline.clarity.scss

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('timeline.clarity') {
+  .clr-timeline {
+    display: flex;
+    padding: $clr-timeline-padding-width;
+  }
+
+  .clr-timeline-step {
+    display: flex;
+    position: relative;
+
+    clr-icon {
+      width: $clr-timeline-icon-size;
+      height: $clr-timeline-icon-size;
+      min-width: $clr-timeline-icon-size;
+      min-height: $clr-timeline-icon-size;
+    }
+
+    clr-icon[shape='spinner'] {
+      animation: spin 1s linear infinite;
+    }
+
+    clr-icon[shape='circle'] {
+      color: $clr-gray;
+    }
+
+    clr-icon[shape='dot-circle'] {
+      color: $clr-action-blue;
+    }
+
+    clr-icon[shape='success-standard'] {
+      color: $clr-green;
+    }
+
+    clr-icon[shape='error-standard'] {
+      color: $clr-red;
+    }
+  }
+
+  .clr-timeline-step-header {
+    color: $clr-timeline-step-header-color;
+    font-size: $clr-timeline-step-header-font-size;
+    line-height: $clr-timeline-step-header-line-height;
+  }
+
+  .clr-timeline-step-title {
+    color: $clr-timeline-step-title-color;
+    font-size: $clr-timeline-step-title-font-size;
+    font-weight: $clr-timeline-step-title-font-weight;
+    line-height: $clr-timeline-step-title-line-height;
+    margin-bottom: $clr-timeline-step-title-padding-bottom;
+  }
+
+  .clr-timeline-step-description {
+    color: $clr-timeline-step-description-color;
+    font-size: $clr-timeline-step-description-font-size;
+    line-height: $clr-timeline-step-description-line-height;
+    button {
+      display: block;
+      margin-top: $clr-timeline-step-description-button-top-margin;
+    }
+    img {
+      width: 100%;
+      margin-top: $clr-timeline-step-description-img-top-margin;
+    }
+  }
+
+  // specific styles for horizontal timeline
+  .clr-timeline.clr-timeline-horizontal {
```

Comment:
I think we should assume horizontal is default, so they shouldn't have to add `.clr-timeline-horizontal` to achieve that.

Reply:
After taking a look at some other component at Clarity, I agree it's the right solution to actually default to horizontal. I'll work on merge the styles.